### ISSUE TITLE
日常Issues交流

### ISSUE THREAD

**Issue by jiaxiaogang** [jiaxiaogang/he4o#4]
## 非常感谢

```
NSLog(@"Thanks very much!");
```

**Comment by zengcity** [comment]
为啥选择OC呢？

**Comment by hechzh** [comment]
目前有确实的demo吗？有训练效果吗？

**Comment by SeanChense** [comment]
这个具体能做什么呢

**Comment by DaidoujiChen** [comment]
希望可以多一些簡單點的範例 =w=

**Comment by zhoujiuhai** [comment]
文档太天马行空了~

**Comment by jiaxiaogang** [comment]
@hechzh 这个源码是可以下载运行的,不过目前进度才20%左右;尚未能达到出众的效果;另外SMG_NOTE和PR_DBHelper两个项目也是与此相关的两个;一个是笔记一个是基于这个项目中的神经网络开发的数据方案;你可以稳步看看;谢谢支持;

**Comment by jiaxiaogang** [comment]
@SeanChense SMG是一个强人工智能系统项目,主要是研究人工智能;并将其与计算机软件进行融合;感兴趣的话可以看看我的笔记 <github.com/jiaxiaogang/SMG_NOTE>代码我也在每天跟进中,谢谢关注;

**Comment by jiaxiaogang** [comment]
@DaidoujiChen 对,有一些小的示例,但小的示例并不能展现出全面的智能,只能展示出其一部分功能,你可以看看< github.com/jiaxiaogang/PR_DBHelper> 和< github.com/SMGTeam/Log >这两个项目;都是基于此项目的小扩展;谢谢支持..^_^;

**Comment by jiaxiaogang** [comment]
文档确实有点乱,有朋友正在帮我整理并出一本书,请稍等时日;另外如果有问题,可以直接与我在github中点击绿色按钮new Issues交流;谢谢支持;


发送自 Outlook<http://aka.ms/weboutlook>

________________________________
发件人: Joe Chow <notifications@github.com>
发送时间: 2017年8月24日 10:03
收件人: jiaxiaogang/SMG
抄送: 赤臂生; Author
主题: Re: [jiaxiaogang/SMG] 大家可以在此问问我一些问题;谢谢; (#4)


文档太天马行空了~

―

**Comment by oksimple** [comment]
基于什么语言实现的？

**Comment by jiaxiaogang** [comment]
> 目前有确实的demo吗？有训练效果吗？

可以看：https://zhuanlan.zhihu.com/p/48730039
目前的demo较简单，在文中有简要描述。

可以全面体现he4o实力的demo“小鸟生存演示”，已在开发中，届时再发出来。
（不过如果对he4o感兴趣，现在就该行动）

**Comment by jiaxiaogang** [comment]
> 基于什么语言实现的？

oc，团队化后。会改由c来开发与迭代。（一个系统的内核，效率第一）

**Comment by jiaxiaogang** [comment]
> 这个具体能做什么呢

agi的用途比较广泛，不过我做的并不是应用层，所以在这方面的想法比较通俗，无法很好回答你的问题。
